app/api/templates/templates.ts

```diff
@@ -304,5 +304,9 @@ export default {
     return model.count({ 'properties.content': thesauriId });
   },
 
+  async countByRelationType(relationTypeId: string) {
+    return model.count({ 'properties.relationType': relationTypeId });
```

Comment:
It might be a bit much, but since we have collections with multiple templates that can have over a 100 properties, maybe adding an index for this on the model might be worth it, since it is not a big effort.

Reply:
100 is not, IMO, a big number. Also, the deletes of relationship types shouldn't be frequent. I would say it doesn't justify the penalty of maintaining the index.